connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/ClientE2ETestSuite.scala

```diff
@@ -631,6 +631,19 @@ class ClientE2ETestSuite extends RemoteSparkSession with SQLHelper {
     val otherPlan = spark.sql("select 1")
     assert(plan.sameSemantics(otherPlan))
   }
+
+  test("toJSON") {
+    val expected = Array(
+      "{\"b\":0.0,\"id\":0,\"d\":\"world\",\"a\":0}",
```

Comment:
Use raw strings here. That is easier to read.
```scala
val expected = Array(
  """{"b": 0.0,"id":0,"d":"world","a":0}""",
  ...
)
```

Reply:
done